azkaban-common/src/main/java/azkaban/executor/ExecutableRamp.java

```diff
@@ -365,12 +369,27 @@ synchronized public void cacheResult(Action action) {
             / ((this.state.numOfTrail + this.state.cachedNumOfTrail) * 1.0))
         : (this.state.numOfFailure + this.state.cachedNumOfFailure);
 
+    LOGGER.info(String.format("Cache Ramp Result : [id = %s, action: %s, %s failure: %d, numOfTrail (%d, %d), numOfSuccess: (%d, %d), numOfFailure: (%d, %d), numOfIgnore: (%d, %d)]"
```

Comment:
Indentation is not correct. If you have follow the formatting guidelines (in Azkaban github page) it should fix it for you.
Also, break thing into multiple lines as per the guidelines.